datasets/code_refinement/code_refinement.py

```diff
@@ -0,0 +1,179 @@
+        "test_fix": "https://raw.githubusercontent.com/microsoft/CodeXGLUE/main/Code-Code/code-refinement/data/medium/test.buggy-fixed.fixed",
+        "train_bug": "https://raw.githubusercontent.com/microsoft/CodeXGLUE/main/Code-Code/code-refinement/data/medium/train.buggy-fixed.buggy",
+        "valid_bug": "https://raw.githubusercontent.com/microsoft/CodeXGLUE/main/Code-Code/code-refinement/data/medium/valid.buggy-fixed.buggy",
+        "test_bug": "https://raw.githubusercontent.com/microsoft/CodeXGLUE/main/Code-Code/code-refinement/data/medium/test.buggy-fixed.buggy",
+    },
+}
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class NewDataset(datasets.GeneratorBasedBuilder):
+    """TODO: Short description of my dataset."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="code_refinement_small",
+            description="small version of the code_refinement dataset",
+        ),
+        datasets.BuilderConfig(
+            name="code_refinement_medium",
+            description="medium version of the code_refinement dataset",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "code_refinement_small"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+
+    def _info(self):
+        # TODO: This method pecifies the datasets.DatasetInfo object which contains informations and typings for the dataset
+        if (
+            self.config.name == "code_refinement_small"
+        ):  # This is the name of the configuration selected in BUILDER_CONFIGS above
+            features = datasets.Features(
+                {
+                    "bug": datasets.Value("string"),
+                    "fix": datasets.Value("string"),
+                }
+            )
+        else:  # This is an example to show how to have different features for "first_domain" and "second_domain"
+            features = datasets.Features(
+                {
+                    "bug": datasets.Value("string"),
+                    "fix": datasets.Value("string"),
+                }
+            )
```

Comment:
```suggestion
        features = datasets.Features(
            {
                "bug": datasets.Value("string"),
                "fix": datasets.Value("string"),
            }
        )
```
They're the same, no need to check for the config name